src/pages/home.jsx

```diff
@@ -0,0 +1,170 @@
+import React, { useState, useEffect, useRef } from 'react';
+import Cardlist from '../molecules/cardlist';
+import movieAPI from '../atoms/movieAPI';
+import PageLoading from '../atoms/pageLoading';
+import PageChange from '../Redux/actions/PageChange';
+import ResultsChange from '../Redux/actions/ResultsChange';
+import store from '../Redux/store';
+
+const Home = () => {
```

Comment:
This file is a bit messy in the markup section, could be broken up into a couple atoms and molecules to increase readability.